app/views/content/life-as-a-teacher/art-and-design.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Become an art and design teacher
+title_paragraph: |-
+  <p>
+  Teaching art and design opens up a world of creativity.  When pupils create, they learn to take risks, embrace mistakes, and persevere to bring their ideas to life.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_artanddesign$ are available for eligible trainee art and design teachers.</p>
+description: |-
+    Find out how to become an art and design teacher, including what you'll be teaching and what funding is available to help you train.
+subcategory: What to teach
```

Comment:
This needs to be removed, as this page should be in the Explore subjects section now